<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetPGDatabaseSettingDlg</class>
 <widget class="QDialog" name="SetPGDatabaseSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Settings</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostEdt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="portEdt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbEdt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Database User</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="dbUserEdt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="pwdEdt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>190</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="appUserEdt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>230</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>App User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearDataBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Data</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearLogBtn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Log</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearSettingBtn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>270</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Setting</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
